lib/logstash/outputs/kusto/ingestor.rb

```diff
@@ -90,7 +90,8 @@ def upload(path, delete_on_success)
       #   local_ingestion_properties.addJsonMappingName(mapping)
       # end
 
-      @kusto_client.ingestFromSingleFile(path, @ingestion_properties)
+      file_source_info = Java::com.microsoft.azure.kusto.ingest.source.FileSourceInfo.new(path, 0); # 0 - let the sdk figure out the size of the file
```

Comment:
Unrelated, but wouldn't we want to add an overload that doesn't accept the file size to the Java client?

Reply:
Yes, I guess we can do that. Do you have it on other SDKs (Python)?